test/units/modules/network/netvisor/test_pn_dscp_map_pri_map.py

```diff
@@ -0,0 +1,61 @@
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+from units.compat.mock import patch
+from ansible.modules.network.netvisor import pn_dhcp_map_pri_map
```

Comment:
Typo, this looks like it should be `pn_dscp_map_pri_map` here and in several places below.